handler.py

```diff
@@ -38,17 +38,18 @@ def get_memberships(group_id, token):
     response = requests.get(f'{API_ROOT}groups/{group_id}', params={'token': token}).json()['response']['members']
     return response
             
-def get_membership_id_and_admin_status(group_id, user_id, token):
+def get_membership_id(group_id, user_id, token):
```

Comment:
Basically just renamed this function because it doesn't get the admin status anymore (it never did)